packages/web/components/WelcomePlayer.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Flex, Image, Text } from '@metafam/ds';
+import React from 'react';
+
+import PlayersImage from '../public/images/players.png';
+
+type Props = {
+  next: () => void;
+};
+
+export const WelcomePlayer: React.FC<Props> = ({ next }) => {
+  return (
+    <Flex
+      alignItems="center"
+      justifyContent="center"
+      flexDirection="column"
+      flex={1}
+      maxWidth="45rem"
+    >
+      <Image src={PlayersImage} />
+      <Text
+        fontSize="2xl"
+        fontFamily="mono"
+        fontWeight="bold"
+        color="white"
+        m={5}
+      >
+        Become a Player
+      </Text>
+      <Text fontSize="lg" fontFamily="body" color="white" textAlign="center">
```

Comment:
Isnt body the fontFamily by default? Shouldnt need to set it. Same with color, use DarkMode